CONTRIBUTING.md

```diff
@@ -33,16 +39,28 @@ $ git clone git@github.com:<user>/kubebuilder.git $GOPATH/src/sigs.k8s.io/kubebu
 1. Build the project by using the command `make build` 
 1. Run the command `make install` to create a bin with the source code 
 
-## How to test kubebuilder locally
+**NOTE** In order to check the local environment run `make go-test`.
+
+## What to do before submitting a pull request 
 
-1. Run the tests by using the command `make test`. It will execute unit tests. 
-1. Run the script `make generate` to update/generate the mock data used in the e2e test in `$GOPATH/src/sigs.k8s.io/kubebuilder/testdata/`
+1. Run the script `make generate` to update/generate the mock data used in the e2e test in `$GOPATH/src/sigs.k8s.io/kubebuilder/testdata/` 
+2. Run `make lint` to check your code.
+
+**NOTE** To run `make lint` is required to install `golangci-lint` locally. More info: https://github.com/golangci/golangci-lint#install
 
 **IMPORTANT:** The `make generate` is very helpful. By using it, you can check if good part of the commands still working successfully after the changes. Also, note that its usage is a pre-requirement to submit a PR.
 
-**NOTE** To run `make lint` is required to install `golangci-lint` locally it. More info: https://github.com/golangci/golangci-lint#install
+Following the targets that can be used to test your changes locally.
+
+|   Command	|   Description	|  Is called in the CI?  	|
+|---	|---	|---	|
+| make go-test |  Runs go tests | no   	|
```

Comment:
I think this test is being called inside `./test.sh` and `./check_testdata.sh` so technichally the CI is running it.

Reply:
No. It is not. See; 

https://github.com/kubernetes-sigs/kubebuilder/blob/1433321257750fde2bec5e34ab6c5b7115f5dafa/Makefile#L71-L73

Reply:
Yes, it is. See:
https://github.com/kubernetes-sigs/kubebuilder/blob/7a259a399a5ef2e072f5b962a810968fd2cbdd4b/test.sh#L157

Reply:
ah I catcher what you mean now. it is over `Is called in the CI?`. 
Ok, but the target is not called used in the CI. So, I think it still false and true for the targets that are calling  ./test.sh and ./check_testdata.sh explicit.